tests/unit/repo/experiments/queue/test_local.py

```diff
@@ -1,5 +1,13 @@
+import time
+from collections import namedtuple
+
+import pytest
 from celery import shared_task
 
+from dvc.repo.experiments.exceptions import UnresolvedExpNamesError
+
+Entry = namedtuple("Entry", ["stash_rev"])
```

Comment:
Instead of defining this test only type you can use the `Mock()` type and set the specific attributes you need. So in these tests you could do

```py
mock_queue_entry = mocker.Mock(stash_rev="...")
```

This makes it a bit more obvious that the object is a mocked/testing only object.

Alternatively, if you are going to define test only types it's better to name them with the `_` prefix (like `_Entry`) so that it's obvious it's an internal testing type specific to this module (and not something we are importing from DVC code)

Reply:
Current PR LGTM otherwise